README.md

```diff
@@ -49,34 +49,31 @@ Our builds can be fetched from `kinvolk.io/aci/rkt/stage1-kvm`, e.g.
 rkt image fetch --insecure-options=image kinvolk.io/aci/rkt/stage1-kvm:1.23.0,kernelversion=4.9.6
 ```
 
-## Usage on Semaphore CI
+## Using custom built stage1-kvm images on Semaphore CI
 
-As Semaphore CI doesn't include rkt by default on their platform, rkt must be
-downloaded as a first step.
+See `examples/semaphore.sh` for an example script which shows the necessary
+steps to use a custom stage1-kvm image and serves as a starting point.
 
-Example `semaphore.sh`:
+### Configuration
```

Comment:
the platform must be configured as well: by default it does not use the platform marked with "docker support"